src/mono/wasm/build/WasmApp.targets

```diff
@@ -318,4 +356,10 @@ EMSCRIPTEN_KEEPALIVE void mono_wasm_load_profiler_aot (const char *desc) { mono_
 
     <Exec Command="chmod a+x $(WasmRunV8ScriptPath)" />
   </Target>
+
+  <Target Name="_AfterWasmBuildApp">
+    <ItemGroup>
+      <WasmAssembliesFinal Include="@(_WasmAssemblies)" />
```

Comment:
Should the SDK start using this item group?

Reply:
Yeah, once we merge this